docs/feature_flags.md

```diff
@@ -194,3 +194,15 @@ won't work when you push OTLP metrics.
 
 Enables PromQL functions that are considered experimental and whose name or
 semantics could change.
+
+## Created Timestamps Zero Injection
+
+`--enable-feature=created-timestamp-zero-ingestion`
+
+Enables ingestion of Created Timestamps as injection of samples with 0 as value. 
+
+Created timestamps are only supported by the traditional Prometheus Protobuf protocol. When enabling this feature,
+therefore, the default scrape protocol will be switched to Prometheus Protobuf.
+
+Besides enabling this feature in Prometheus, created timestamps need to be exposed by the application being scraped.
+So far, only Prometheus Go Client has support for it.
```

Comment:
```suggestion
```
Perhaps I would remove this part as it's easy to change soon and we will for sure forget to update this piece of documentation (:

Reply:
yeah fair point